EpikCord/client/websocket_client.py

```diff
@@ -265,7 +308,111 @@ def stop_loop_on_completion(f: asyncio.Future):
             pass
         finally:
             future.remove_done_callback(stop_loop_on_completion)
-            self.utils.cleanup_loop(loop)
+            Utils.cleanup_loop(loop)
+
+
+
+    async def _voice_server_update(self, data: discord_typings.VoiceServerUpdateEvent):
+        voice_data: discord_typings.VoiceServerUpdateEvent = data["d"]
+        payload = {
+            "token": voice_data["token"],
+            "endpoint": voice_data["endpoint"],
+            "guild_id": voice_data["guild_id"],
+        }
+
+        if voice_data["endpoint"]:
+            payload["endpoint"] = voice_data["endpoint"]
+
+        return await self.dispatch("voice_server_update", payload)
+
+    async def _voice_state_update(self, data: discord_typings.VoiceStateUpdateEvent):
+        from EpikCord import VoiceState
+
+        return await self.dispatch(
+            "voice_state_update", VoiceState(self, data)
+        )  # TODO: Make this return something like (VoiceState, Member) or make VoiceState get Member from member_id
+
+    async def _guild_members_chunk(self, data: discord_typings.GuildMembersChunkEvent):
+        ...
```

Comment:
Raise a clear `NotImplementedError` for the end user to be aware of the feature not be implemented rather than doing silently nothing and leave potential problem happening

Reply:
Yep